buildbot.mariadb.org/scripts/docker_cleanup.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+set -xeuvo pipefail
+
+mariadb_version=${1}
+mariadb_version=${mariadb_version#*-}
+master_branch=${mariadb_version%\.*}
+
+commit=${2:-}
+branch=${3:-${master_branch}}
+
```

Comment:
I am wondering if we should not use `getopts` more frequently for those script? It makes them easier to debug and above all more robust in case they are not called correctly:

```bash
  usage() {
    cat >&2 <<-EOF
    Usage : $0 -v <mariadb-version> -c <commit> -b <branch>
      -v mariadb version
      -c commit hash
      -b branch
      -h help
  EOF
  }

  typeset mariadb_version=""
  typeset commit=""
  typeset branch=""

  while getopts "v:c:b:h" OPTION; do
    case $OPTION in
      v)
        mariadb_version="$OPTARG"
        ;;
      c)
        commit="$OPTARG"
        ;;
      b)
        branch="$OPTARG"
        ;;
      h)
        usage
        exit 0
        ;;
      *)
        usage
        exit 1
        ;;
    esac
  done

  [[ $mariadb_version != "" ]] || {
    usage
    exit 1
  }
  [[ $commit != "" ]] || {
    usage
    exit 1
  }
  [[ $branch != "" ]] || {
    usage
    exit 1
  }

  mariadb_version=${mariadb_version#*-}
  master_branch=${mariadb_version%\.*}
  commit=${$commit:-}
  branch=${$branch:-${master_branch}}
```